web/src/components/InputWithButton/index.tsx

```diff
@@ -0,0 +1,71 @@
+import '@openware/cryptofont';
+import classnames from 'classnames';
+import * as React from 'react';
+import { Button, InputGroup } from 'react-bootstrap';
+import { CustomInput } from '../';
+import { MouseEventHandler } from "react";
+
+const InputWithButton = ({
+  value,
+  className,
+  fieldId = '',
+  disabled = false,
+  readOnly = false,
+  buttonText = '',
+  label = '',
+  type = "text",
+  handleClickInput = () => {},
+  handleClickButton = () => {},
+  handleChangeInput = () => {},
+}: {
+  value: any;
+  className: string;
+  fieldId?: any;
+  disabled?: boolean;
+  buttonText?: string;
+  label?: string;
+  type?: string;
+  readOnly?: boolean;
+  handleClickInput?: MouseEventHandler;
+  handleClickButton?: MouseEventHandler;
+  handleChangeInput?: (value: string) => void;
+}) => {
+  const cx = classnames('cr-copyable-text-field', className);
```

Comment:
since now InputWithButton is parent component for Copyable text field you should remove `cr-copyable-text-field` class for InputWithButton and create its own className